Contributing.md

```diff
@@ -14,37 +13,29 @@ Thank you for your interest in our documentation!
 Here are some ways you can contribute to this documentation:
 
 * To make small changes to an article, [Contribute using GitHub](#contribute-using-github).
-* To make large changes, or changes that involve code, [Contribute using Git](#contribute-using-git).
-* Report documentation bugs and request new documentation via GitHub Issues.
+* Report documentation bugs and request new documentation via [GitHub Issues](/issues).
 
-## Contribute using GitHub
+## Using Git and GitHub
 
-> **Important**: The reference content in this repository is autogenerated. Pull requests to this repo should address tooling or other infrastructure issues.
+**Note:** Most of the information in this section can be found in [GitHub Help](https://docs.github.com/) articles. If you're familiar with Git and GitHub, skip to the [Adding samples](#adding-samples) section for the specifics of the code/content flow of this repository.
 
-Use GitHub to contribute to this documentation without having to clone the repo to your desktop. This is the easiest way to create a pull request in this repository. Use this method to make a minor change that doesn't involve code changes.
+Use GitHub to contribute to this documentation without having to clone the repo to your desktop. This is the easiest way to create a pull request in this repository. It's the recommended way to add a single sample.
 
-Using this method allows you to contribute to one article at a time.
+> **Important**: The reference content in this repository is autogenerated. Pull requests to this repo should either add samples or address tooling or other infrastructure issues.
```

Comment:
The preceding "Note:" on (new) line 20 has the colon inside the asterisks. Recommending that the colon should be in the same location (both inside the asterisks, or both outside) in both instances, although I don't have a strong opinion about which location. 
```suggestion
> **Important:** The reference content in this repository is autogenerated. Pull requests to this repo should either add 
samples or address tooling or other infrastructure issues.
```

Reply:
It should be outside. That's consistent with our other documentation.